x/programs/runtime/import_program.go

```diff
@@ -19,6 +18,12 @@ const (
 	getFuelCost     = 10000
 )
 
+var (
+	ExecutionFailure = []byte{0}
+	CallPanicked     = []byte{1}
+	OutOfFuel        = []byte{2}
```

Comment:
These shouldn't be `[]byte`, they should just be `u8` that are inserted into a `[]byte` when necessary.

Reply:
Replaced all of them by constants of ints, the `[]byte` value is constructed at runtime now.